server/src/main/kotlin/fi/oph/kitu/yki/YkiApiController.kt

```diff
@@ -16,6 +17,7 @@ import java.io.ByteArrayInputStream
 class YkiApiController(
     private val service: YkiService,
 ) {
+    @WithSpan
```

Comment:
Olin luulossa että `GetMapping`-annotoidut (ja muut spring-web-annotoidut metodit) saisivat automaattisesti spanin muutenkin, eli tässä ei tarvisi WithSpania. Olenko ihan väärässä? Ref: https://opentelemetry.io/docs/zero-code/java/spring-boot-starter/out-of-the-box-instrumentation/#spring-web-mvc-autoconfiguration